robyn/logger.py

```diff
@@ -0,0 +1,77 @@
+from enum import Enum
+import logging
+from typing import Optional
+
+
+class Colors(Enum):
+    BLUE = "\033[94m"
+    CYAN = "\033[96m"
+    GREEN = "\033[92m"
+    YELLOW = "\033[93m"
+    RED = "\033[91m"
+
+
+HEADER = "\033[95m"
+ENDC = "\033[0m"
+BOLD = "\033[1m"
+UNDERLINE = "\033[4m"
```

Comment:
you can move them inside the `Logger` class like
```
class Logger:
     HEADER = "\033[95m"
     ENDC = "\033[0m"
     BOLD = "\033[1m"
     UNDERLINE = "\033[4m"
```

Reply:
Yes that makes sense